lib/FrontendTool/ScanDependencies.cpp

```diff
@@ -125,12 +125,15 @@ static std::vector<ModuleDependencyID> resolveDirectDependencies(
                                   knownModules);
     }
 
-    // Look for overlays for each of the Clang modules. The Swift module
+    // Look for overlays for each of the Clang modules. The Swift module
```

Comment:
```suggestion
    // Look for overlays for each of the Clang modules. The Swift module
```

Reply:
Thanks for the catch. I need to figure out why my Xcode keeps inserting these in strange places.